server/controllers/index.js

```diff
@@ -0,0 +1,6 @@
+import UserController from './user.controller';
+
+export {
+  // eslint-disable-next-line import/prefer-default-export
```

Comment:
Why are we disabling this eslint rule?

Reply:
While following the agreed folder structure, we experienced issues using `export default` on that line and ESLint didn't allow only `export {..}`. That's why we resolved to disabling it

Reply:
I was able to find a work around this.